eventservice/api/controllers/events.py

```diff
@@ -312,26 +314,15 @@ def put(event_id):
         __logger.exception(msgs.ERR_MSG_UPDATE)
         abort(400)
 
-    group_memberships = list()
-    try:
-        _, group_memberships = get_group_memberships()
-    except Exception as ex:
-        __logger.exception(msgs.ERR_MSG_GET_GROUP_MEMBERSHIP)
-        abort(500)
-    db = None
-    event = None
-    try:
-        db = get_db()
-        event = db['events'].find_one({'_id': ObjectId(event_id)}, {'_id': 0})
-    except Exception as ex:
-        __logger.exception(msgs.ERR_MSG_GET_EVENT)
-        abort(500)
-
-    # If this is a group event, apply group authorization. Regular events can proceed like before.
-    if not check_group_event_admin_access(event, group_memberships):
-        abort(401)
+    # check group event
+    if req_data.get('createdByGroupId'):
```

Comment:
After rethinking, I think we should use the event data from the database and not the incoming request to decide whether this is a group event or not.

In the rare case of changing a group event to a non-group event, the request will not contain the createdByGroupId, it will go to the auth_middleware.authorize case, which we probably don't want. Any thoughts?